manageiq-operator/pkg/helpers/miq-components/util.go

```diff
@@ -61,6 +61,13 @@ func addAppLabel(appName string, meta *metav1.ObjectMeta) {
 	meta.Labels["app"] = appName
 }
 
+func addBackupLabel(backupLabel string, meta *metav1.ObjectMeta) {
+	if meta.Labels == nil {
+		meta.Labels = make(map[string]string)
+	}
+	meta.Labels[backupLabel] = "t"
```

Comment:
Why `t` as opposed to `true` or an actual boolean?

Reply:
You have to use a string as the label value.  Things were getting confused when I put `true` there because it thought i was putting a boolean in a string field.

Reply:
Ok, I don't love it, but let's merge.